src/deadline/keyshot_submitter/Submit to AWS Deadline Cloud.py

```diff
@@ -0,0 +1,386 @@
+                                        "output_format: 'RENDER_OUTPUT_{{Param.OutputFormat}}'\n"
+                                        ""
+                                    ),
+                                }
+                            ],
+                            "actions": {
+                                "onEnter": {
+                                    "command": "KeyShotAdaptor",
+                                    "args": [
+                                        "daemon",
+                                        "start",
+                                        "--path-mapping-rules",
+                                        "file://{{Session.PathMappingRulesFile}}",
+                                        "--connection-file",
+                                        "{{Session.WorkingDirectory}}/connection.json",
+                                        "--init-data",
+                                        "file://{{Env.File.initData}}",
+                                    ],
+                                    "cancelation": {"mode": "NOTIFY_THEN_TERMINATE"},
+                                },
+                                "onExit": {
+                                    "command": "KeyShotAdaptor",
+                                    "args": [
+                                        "daemon",
+                                        "stop",
+                                        "--connection-file",
+                                        "{{ Session.WorkingDirectory }}/connection.json",
+                                    ],
+                                    "cancelation": {"mode": "NOTIFY_THEN_TERMINATE"},
+                                },
+                            },
+                        },
+                    }
+                ],
+                "script": {
+                    "embeddedFiles": [
+                        {
+                            "name": "runData",
+                            "filename": "run-data.yaml",
+                            "type": "TEXT",
+                            "data": "frame: {{Task.Param.Frame}}\n",
+                        }
+                    ],
+                    "actions": {
+                        "onRun": {
+                            "command": "KeyShotAdaptor",
+                            "args": [
+                                "daemon",
+                                "run",
+                                "--connection-file",
+                                "{{ Session.WorkingDirectory }}/connection.json",
+                                "--run-data",
+                                "file://{{ Task.File.runData }}",
+                            ],
+                            "cancelation": {"mode": "NOTIFY_THEN_TERMINATE"},
+                        }
+                    },
+                },
+            }
+        ],
+    }
+
+
+def construct_asset_references(settings: Settings) -> dict:
```

Comment:
nit: return value typing can be expanded, looks like there are a few other places as well that I won't leave a comment on each one.